test/js/cliSpec.js

```diff
@@ -298,4 +300,87 @@ describe('Tests for cli', () => {
 			expect(consoleError).to.eql([errorMessage]);
 		});
 	});
+	describe('create webhooks tests', function () {
+		const createWebhookResponse = require('../resources/github/createWebhookResponse.json');
+		const httpResponse = {
+			status: 200,
+			data: createWebhookResponse
+		};
+		let createWebhookStub;
+		before(() => {
+			migrateStub = sinon.stub(migrate, 'createWebhook');
+			createWebhookStub = function StubMigrate() {
+				this.createWebhook = migrateStub;
+			};
+		});
+		after(() => {
+			sinon.restore();
+		});
+
+		it('should create webhook', async () => {
+			//given
+			const configFile = 'test/resources/github/webhookTemplate.yml';
+			const events = [
+				'push',
+				'pull_request'
+			];
+			const secret = 'some-secret';
+			const payloadUrl = 'https://github-webhook-proxy/webhook?targetUrl=https://jenkins.some-jenkins.com/github-webhook/';
+			const orgName = 'some-org';
+			const repoName = 'some-repo';
+			migrateStub.returns(httpResponse);
```

Comment:
Lets not expose http response outside of gitlabClient. Client should be the only one responsible for talking in HTTP terms, rest everywhere could be plain JS objects

Reply:
done